project/src/main/kotlin/org/ionproject/core/common/siren.kt

```diff
@@ -6,11 +6,11 @@ import org.springframework.web.util.UriTemplate
 import java.net.URI
 
 class Relation(
-    val rel: String,
-    val href: URI,
-    val type: String? = null,
-    val title: String? = null,
-    @JsonProperty("class") val klass: List<String>? = null)
+        val rel: MutableList<String>,
```

Comment:
For consistency, change this to an immutable type. SirenBuilder is currently the only type capable of having mutable properties.